packages/calypso-e2e/src/lib/pages/plugins-page.ts

```diff
@@ -71,4 +78,48 @@ export class PluginsPage {
 		await this.page.click( selectors.breadcrumb( text ) );
 		assert.match( this.page.url(), /.*\/plugins$/ );
 	}
+
+	/**
+	 * Click Monthly Pricing
+	 */
+	async clickMonthlyPricing(): Promise< void > {
+		await this.page.click( selectors.monthlyPricingToggle );
+	}
+
+	/**
+	 * Click Annual Pricing
+	 */
+	async clickAnnualPricing(): Promise< void > {
+		await this.page.click( selectors.annualPricingToggle );
+	}
+
+	/**
+	 * Select monthly
+	 */
+	async selectMonthlyPricing(): Promise< void > {
+		await this.page.click( selectors.pricingToggle );
+		await this.page.click( selectors.monthlyPricingSelect );
+	}
+
+	/**
+	 * Select annual
+	 */
+	async selectAnnualPricing(): Promise< void > {
+		await this.page.click( selectors.pricingToggle );
+		await this.page.click( selectors.annualPricingSelect );
+	}
+
+	/**
+	 * Check Is Monthly Pricing
+	 */
+	async checkMonthlyPricing(): Promise< void > {
```

Comment:
This is an excellent way to do validating when you need to wait for some kind of state change in the DOM! 🎉 

Tiny style nit (that we haven't documented, so our bad): we typically prefix these kinds of methods with `validate`.

Reply:
Thanks, will update.